ui/app/hooks/tests/useCancelTransaction.test.js

```diff
@@ -0,0 +1,105 @@
+import * as reactRedux from 'react-redux'
+import assert from 'assert'
+import { renderHook } from '@testing-library/react-hooks'
+import sinon from 'sinon'
+import transactions from '../../../../test/data/transaction-data.json'
+import { getConversionRate, getSelectedAccount } from '../../selectors'
+import { useCancelTransaction } from '../useCancelTransaction'
+import { showModal } from '../../store/actions'
+import { increaseLastGasPrice } from '../../helpers/utils/confirm-tx.util'
+
+let useSelector, useDispatch
+
+const dispatch = sinon.spy()
```

Comment:
Can we move this into the describe block scope instead of module scope and maybe also use a [sandbox](https://sinonjs.org/releases/v9.0.2/sandbox/) to simplify the restoration?